ansibullbot/utils/gh_gql_client.py

```diff
@@ -227,7 +227,7 @@ def get_summaries(self, owner, repo, otype='issues', last=None, first='first: 10
                 u'variables': u'{}',
                 u'operationName': None
             }
-            rr = requests.post(self.baseurl, headers=self.headers, data=json.dumps(payload))
+            rr = requests.post(self.baseurl, headers=self.headers, data=json_dumps(payload))
```

Comment:
From this old bug: https://github.com/requests/requests/issues/3875 you may have to use to_bytes() with this as well.  I'm not certain, though.

Reply:
Well, requests methods support `json=` arg so it's better just use it then:
```suggestion
            rr = requests.post(self.baseurl, headers=self.headers, json=payload)
```

Reply:
Sounds good to me.